genres/genres_service.go

```diff
@@ -20,13 +20,12 @@ func NewCypherGenresService(cypherRunner neoutils.CypherRunner, indexManager neo
 
 func (s service) Initialise() error {
 	return neoutils.EnsureConstraints(s.indexManager, map[string]string{
-		"Thing":             "uuid",
-		"Concept":           "uuid",
-		"Classification":    "uuid",
-		"Genre":             "uuid",
-		"FactsetIdentifier": "value",
-		"TMEIdentifier":     "value",
-		"UPPIdentifier":     "value"})
+		"Thing":          "uuid",
+		"Concept":        "uuid",
+		"Classification": "uuid",
+		"Genre":          "uuid",
+		"TMEIdentifier":  "value",
+		"UPPIdentifier":  "value"})
```

Comment:
That's the identifier for all of our uuids, canonical and none canonical (so the UPP platform uuids, that we will use to search for an object - it makes more sense for concorded organisations, but we adopted the same pattern everywhere).